test/errors/unknownEnumElement.chpl

```diff
@@ -0,0 +1,5 @@
+enum creativeColors {
+  red, blue
```

Comment:
Those are not very creative!  :D  (I was expecting to find things like 'chartreuse' and 'teal' here :D ).

Reply:
Perhaps :D I was merely adapting the usual color enum example with a [little bit of a reference](https://youtu.be/9C_HReR_McQ?t=105)

Reply:
Definitely went right over this old head!  Thanks for filling in the blanks...